package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "preinstall": "node install-k6.js",
     "start": "electron-forge start",
+    "start:extension": "STANDALONE_EXTENSION=true vite --config vite.extension.config.mts",
```

Comment:
This lets us start a browser with the extension installed without running k6 Studio. Handy when you're more focused on the extension side of things.